src/qrcode/QRTabBar.tsx

```diff
@@ -46,6 +46,10 @@ export default function QRTabBar({ state, descriptors, navigation, position, qrS
     dispatch(shareQRCode(qrSvgRef.current))
   }
 
+  const getParams = (route: string) => {
```

Comment:
Previously, params were discarded when a user switched tab and switched back again.